Source/SelfService/Web/microservice/microserviceView/configuration/filesSection/buttonGroup.tsx

```diff
@@ -34,7 +34,7 @@ export const ButtonGroup = ({ filePrompt, deleteDisabled, downloadDisabled, hand
 
         <Button
             variant='text'
-            label={`Download Configuration Maps`}
+            label={`Download files yaml`}
```

Comment:
The button text here feels wrong. The yaml that is downloaded is a `ConfigMap, [as you can see here on dolittle.io](https://dolittle.io/docs/platform/update_configurations/). I know we probably want to move away from this, but perhaps a better wording would be: 
```suggestion
            label={`Download Configuration Files Yaml`}
```